### PR TITLE
[RFR] Italian translation

### PR DESCRIPTION
**Pull request by HugoGiraudel**

### CODE REVIEW

File: it/_contributing.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+# Contributi
+
+"Linee guida di Sass" è un progetto libero di cui mi prendo cura nel mio tempo libero. Inutile a dirsi, c'è un bel po' di lavoro da fare per tenere tutto aggiornato, documentato e rilevante.
```
Comment (HugoGiraudel) [inline it/_contributing.md]: Any reason you did not keep “Sass Guidelines” at the beginning of the sentence?

Comment (mattcav) [inline it/_contributing.md]: Even if "linee guida di Sass" is a matching translations for Sass guidelines, makes sense keeping the original title here.
I will correct it here: https://github.com/HugoGiraudel/sass-guidelines/pull/239

**Comment by mattcav** [comment]
I think you can close this one and move here: https://github.com/HugoGiraudel/sass-guidelines/pull/239

**Comment by mattcav** [comment]
ok @HugoGiraudel, I think we got it.

**Comment by HugoGiraudel** [comment]
The `it/tldr.md` file seems to be missing entirely. :(

**Comment by HugoGiraudel** [comment]
FUCK YEEEEEES! :tada:

**Comment by HugoGiraudel** [comment]
Thank you so much guys. :)

**Pull request merged**